cmd/zt_copy_blob_download_test.go

```diff
@@ -422,8 +422,10 @@ func (s *cmdIntegrationSuite) TestDownloadBlobContainerWithPattern(c *chk.C) {
 
 	// construct the raw input to simulate user input
 	rawContainerURLWithSAS := scenarioHelper{}.getRawContainerURLWithSAS(c, containerName)
-	rawContainerURLWithSAS.Path += "/*" // imply strip-top-dir
-	raw := getDefaultCopyRawInput(rawContainerURLWithSAS.String(), dstDirName)
+	urlString := rawContainerURLWithSAS.String()
+	// imply strip-top-dir (without the /* getting escaped by the line above)
+	urlString = strings.Replace(urlString, "?", "/*?", -1) // TODO: find a better way to do this in our tests
```

Comment:
>"/*?" [](start = 45, length = 5)

I'm sure this works, but our rawContainerURLWithSAS will never end with a /, right? If it does, we'd accidentally end up traversing account.blob.core.windows.net/container//*?SAS

Reply:
For now, given the way rawContainerURLWithSAS works, we're fine. I'll let the existing TODO remind us to find something better.

Reply:
Will leave it covered by the existing TODO for now.  Agree we should make it more robust against change in the generantion of the URL.